PyDSS/value_storage.py

```diff
@@ -60,13 +60,39 @@ def to_dataframe(self):
 
         """
 
+class   ValueByList(_ValueStorageBase):
```

Comment:
What is the difference between this class and ValueByLabel? Could you change the docstring so that a reader can understand right away?
Also, you have some extra spaces in there.

Reply:
Some function calls return lists. **see transfomer_taps output file.** 
this was implemented to resolve such issues.

Reply:
done